pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -159,8 +160,7 @@ private ResourceUsage getMemUsage() {
     }
 
     private boolean isPhysicalNic(Path path) {
-        try {
-            path = Files.isSymbolicLink(path) ? Files.readSymbolicLink(path) : path;
+            path = Files.isSymbolicLink(path) ? path.toAbsolutePath() : path;
```

Comment:
`readSymbolicLink` can throw `IOException`, but since it was changed, there is no possibility of a checked exception being thrown. I could change it to catch `Exception` instead, though this catches more exceptions than in the original code.